tests/consensus_tests/test_resharding_deferred.py

```
@@ -0,0 +1,381 @@
+        "replicate_shard": {
+            "from_peer_id": from_peer_id,
+            "to_peer_id": to_peer_id,
+            "shard_id": from_shard_id,
+            "to_shard_id": to_shard_id,
+            "method": "resharding_stream_records",
+        }
+    })
+    assert_http_ok(resp)
+
+    time.sleep(1)
+    wait_for_collection_shard_transfers_count(peer_url, COLLECTION_NAME, 0)
```

Ruff (0.15.6)

[error] `wait_for_collection_shard_transfers_count` may be undefined, or defined from star imports

(F405)